roles/engine/vars/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# maintains file name, binary names, and anything which is not a directory or path
+
+# config file names
+engine_config_file_name: Settings.toml
+
+# Environment file names
+engine_environment_general_file_name: "{{ service_identifier }}.env"
+
+# keys name
+engine_node_key_file_name: node_key_file
+engine_signing_key_file_name: signing_key_file
+engine_ethereum_private_key_file_name: ethereum_key_file
+
+# binary and shared libraries names
+engine_engine_default_binary_name: engine-runner
+engine_shared_libraries_default_name:
+    - libchainflip_engine_v1_6_8.so
+    - libchainflip_engine_v1_7_5.so
```

Comment:
this should change with the version, shouldnt it ?

Reply:
yes they changes

Reply:
Thinking of moving them to group_vars

Reply:
yeah lets do it